config/services/buckets-mdapi/service.json.lab

```diff
@@ -1,5 +1,8 @@
 {
 	"params": {
 		"ram": 1024
+        },
+        "metadata": {
+                "BUCKETS_MDAPI_CUEBALL_MAX_CONNECTIONS": 64
```

Comment:
nit: spaces in a file that was using tabs. I'm fine one way or the other, but mixed tabs and spaces is pretty messy.

Reply:
fixed